### DISCUSSION TITLE
"Following" user tab has broken layout

### DISCUSSION THREAD

**Discussion by loup-brun** [community/community#12366] [Other Feature Feedback, Questions, & Ideas]
Go to a user’s **Following** tab. You’ll see the layout is broken there, with duplicate elements and overlapping/fixed positioning of various elements.

![github-following-tab](https://user-images.githubusercontent.com/9596476/156648756-e99bc068-d990-46b0-832c-8fe48ffb79ff.gif)

**Comment by TechSolomon** [comment]
I had a similar issue with GitHub Lists (https://github.com/github/feedback/discussions/12365) -- it looks resolved now.
